pymodels/SI_V25_04/lattice.py

```diff
@@ -365,9 +365,9 @@ def create_lattice(
         L500p, LID3, L500]  # high beta ID straight section (MANACA)
 
     IDB_10 = [
-        L297, L576p, IDQS, L203, IDBPM, L109, IDC1, L218p,
```

Comment:
by removing IDQS and IDC1 should you not update `get_family_data` ?

Reply:
I removed DELTA from the lattice and added EPU again. Next week I will have the information about the new position of DELTA correctors, so I will update the lattice again.